quandl/get_table.py

```diff
@@ -6,7 +6,7 @@
 import os
 
 
-def get_table(code, **options):
+def get_table(datatable_code, **options):
```

Comment:
Then my only suggestion would to be to rename it to `quandl_code` rather than `datatable_code`